backend/src/components/secureExchange.js

```diff
@@ -503,6 +504,24 @@ async function createSecureExchangeStudent(req, res) {
   }
 }
 
+async function removeSecureExchangeStudent(req, res){
+  try {
+    const accessToken = getAccessToken(req);
+    if (!accessToken) {
+      return res.status(HttpStatus.UNAUTHORIZED).json({
+        message: 'No access token'
+      });
+    }
+
```

Comment:
Do we need something like this here? Checks that user has secure_exchange permission
if (!req.session.activeInstitutePermissions.includes('SECURE_EXCHANGE')) {
    return errorResponse(res, 'You are not authorized to access this page.', HttpStatus.UNAUTHORIZED);
  }